pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -152,16 +155,46 @@ func (o *capability) UnregisterFromWorkflow(ctx context.Context, request capabil
 	return nil
 }
 
+// `Execute` enqueues a new consensus request, passing it to the reporting plugin as needed.
+// IMPORTANT: OCR3 only exposes signatures via the contractTransmitter, which is located
+// in a separate process to the reporting plugin LOOPP. However, only the reporting plugin
+// LOOPP is able to transmit responses back to the workflow engine. As a workaround to this, we've implemented a custom contract transmitter which fetches this capability from the
+// registry and calls Execute with the response, setting "method = `methodSendResponse`".
 func (o *capability) Execute(ctx context.Context, r capabilities.CapabilityRequest) (<-chan capabilities.CapabilityResponse, error) {
-	// Receives and stores an observation to do consensus on
-	// Receives an aggregation method; at this point the method has been validated
-	// Returns the consensus result over a channel
-	inputs, err := o.ValidateInputs(r.Inputs)
+	m := struct {
+		Method string
+	}{
+		Method: methodStartRequest,
+	}
+	err := r.Inputs.UnwrapTo(&m)
 	if err != nil {
+		o.lggr.Warnf("could not unwrap method from CapabilityRequest, using default: %w", err)
+	}
+
+	switch m.Method {
+	case methodSendResponse:
+		out := &outputs{
+			WorkflowExecutionID: r.Metadata.WorkflowExecutionID,
+			CapabilityResponse: capabilities.CapabilityResponse{
+				Value: r.Inputs,
```

Comment:
iiuc, r.Inputs contains the newly added Method. if that's correct, why is the entirety of r.Inputs being forwarded in the response? i expected some sort of value extraction that 'excludes the Method header'